tests-scan

```diff
@@ -390,8 +382,18 @@ def cockpit_tasks(api, update, branch_contexts, repo, pull_data, pull_number, sh
                 checkout_ref = ref
                 if project != repo:
                     checkout_ref = "master"
+                    bots_ref = ""
```

Comment:
This looks like a leftover. We shouldn't set it to an empty string, but also it always gets set below.